cmd/utils/flags.go

```diff
@@ -465,6 +465,11 @@ var (
 		Value:    15,
 		Category: flags.PerfCategory,
 	}
+	CacheTrieJournalFlag = &cli.StringFlag{
```

Comment:
Please don't reuse the deprecated flag, as it has slightly different meaning, and also will bring confusion to users.

Let's make another flag called `triedb.journal`

Reply:
It should have a default value, e.g., `DATADIR/triedb.journal` to always enable the file-based journaling